docs/tutorial/chapter2/index.rst

```diff
@@ -39,13 +38,13 @@ format as understood by :py:mod:`configparser`.
 
 Open a new ``helloworld.ini`` in the tutorial directory and copy this into it:
```

Comment:
Is this step still needed?

Reply:
ah because of the rename? yeah, still required because they're not cloning down this folder, they're editing their own project

Reply:
Ah, gotcha!